_sass/components/_icons.scss

```diff
@@ -8,6 +8,21 @@
   width: 3rem;
 }
 
+.icon-leading {
+  width: 1.8rem;
+}
+
+.icon-leading-small {
+  margin-bottom: -3px;
+  width: 1.2rem;
+}
+
+.icon-trailing {
+  margin-bottom: -5px;
+  margin-left: 6px;
+  width: 1.6rem;
+}
+
```

Comment:
I have a couple questions here. 
- I notice that you're mixing `px`, `rem`, and below, there's a class that uses `em`s. Has there been a pattern you're all following for when to use what units, or should we bring these into alignment?
- Should we try to avoid negative margins here through a different approach if there is one? 

Questions probably very relevant for @master12 👀

Reply:
Good points, @jenniferthibault. The `px` units shouldn't affect the responsiveness in this context, but may not be ideal. Positioning these icons is tricky switching from font icons to SVGs. I'll defer to @master12 on better ways to do this.

Reply:
@jenniferthibault I've replaced the `px` units and negative values with a relative option (`vertical-align: middle`). As a result, the back chevron isn't positioned quite as specifically as I was hoping, but it is a better option for responsiveness and maintainability. Thank you!